### ISSUE TITLE
The documentation query returns documentation for value assigned to a property

### ISSUE THREAD

**Issue by simonbs** [ternjs/tern#993]
I'm seeing an issue where the documentation returned by queries of with type `documentation` returns documentation for the value assigned to a property rather than the property. Consider the following examples.

```javascript
let comp = React.Component()
comp.displayName|
```

Assume that the ecmascript.json and react.json definitions have been loaded. When the `end` position corresponds to the position marked by the vertical bar (|), the documentation is returned for the `displayName` property, e.g. "The displayName property is used in debugging messages."

```javascript
let comp = React.Component()
comp.displayName| = "foo"
```

If we assign a string to the `displayName` property and provide the same end position, we get the documentation for the String type, e.g. "The String global object is a constructor for strings..."
This happens no matter which end position within the `displayName` that is provided.

Even if I provide a start and an end position as shown below, the documentation for the String is provided.

```javascript
let comp = React.Component()
comp.|displayName| = "foo"
```

If I remove the ecmascript.json definition and only load react.json, no documentation is provided.

In the above examples, I'm sending the following query to the server.

```json
{
    "query": {
      "type": "documentation",
      "file": fileName,
      "start": start,
      "end": end
    }
}
```

This occurs in version 0.22.2 and 0.22.3. I haven't tested with earlier versions.

**Comment by othree** [comment]
I found this is a definition error.
Will fix soon (also updating the definition of React)

**Comment by othree** [comment]
Fixed by 64926fd
